src/status_im2/contexts/chat/lightbox/top_view.cljs

```diff
@@ -40,8 +41,9 @@
         (anim/animate top-view-bg colors/neutral-100-opa-0)))))
 
 (defn top-view
-  [{:keys [from timestamp]} insets index animations derived landscape? screen-width]
-  (let [display-name                       (first (rf/sub [:contacts/contact-two-names-by-identity
+  [messages insets index animations derived landscape? screen-width]
+  (let [{:keys [from timestamp content]}   (nth @messages @index)
```

Comment:
@OmarBasem, maybe you're aware of what I'm going to say, but `nth` can be unforgiving. If the index is out of bounds it throws an exception and if `@index` is nil, it also throws an exception. So are we sure `@index` can never be nil and that it's always within bounds? Just double-checking.

Reply:
Thanks, I will double check it